teams/2/assets/css/blog_posts.css

```diff
@@ -1,110 +1,209 @@
+    height: 20px;
+    width: 20px;
+    background-size: contain;
+}
+
+.btn-link {
+    color: white;
+    border: 0;
+}
+
+.trial-btns { 
+    display: flex;
+}
+
+.trial-btns button {
+    margin-right: 10px;
+    border: 0;
+    padding: 10px 30px;
+}
+
+.flex-paragraph {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.header-features {
+    background-color: white;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.header-features span,
+.header-features p,
+.header-features h2 {
+    width: 40%;
+    text-align: center;
+    margin: 5px;
+}
+
+.header-features span {
+    color: #6941c6;
+}
+
+.flex-container {
+    background-color: #f5f5f5;
 }
 
 h1 {
     font-size: 50px;
 }
 
 .content {
-    margin-top: 150px;
+    margin-top: 50px;
+    z-index: 20;
 }
 
-.img {
-    background-color: cadetblue;
-    width: 350px;
-    height: 400px;
-    margin-top: 30px;
-    margin-left:-100px;
-    z-index: -1;
-
+.content img {
+    width: 560px;
+    height: 640px;
 }
+
 .flex-container {
     margin-top: 30px;
     display: flex;
     flex-wrap: nowrap;
     justify-content: center;
     justify-content: space-evenly;
-  }
-  .flex-2 {
-    display: grid;
-    justify-content: space-between;
-    grid-template-columns: auto auto auto;
-    margin-left: 50px;
+}
+.flex-2 {
+    display: flex;
+    /* flex wrap */
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
     height: 500px;
     width: 1000px;
-    
-  }
-    .rectangle{
-      width: 900px;
-      height: 200px;
-      background-color: black;
-      justify-content: center;
-      margin-left: 250px;
-    }
-    .rectangle2{
-        width: 900px;
-        height: 200px;
-        background-color: green;
-        justify-content: center;
-        margin-left: 50px;  
-      }
-
-      .rectangle3{
-        width: 500px;
-        height: 200px;
-        background-color: green;
-        justify-content: center;
-        margin-left: 50px;  
-        margin-top: 100px;
-      }
-    .search1{
-        margin-top: 50px;
-        width: 300px;
-        margin-left: 10px;
-    }
-  .flex-container > div {
+    margin: auto;
+}
+.rectangle {
+    width: 900px;
+    height: 200px;
+    background-color: black;
+    justify-content: center;
+    margin-left: 250px;
+}
+.rectangle2 {
+    width: 900px;
+    justify-content: center;
+    margin-left: 50px;
+}
+
+.rectangle3 {
+    width: 500px;
+    height: 200px;
+    background-color: green;
+    justify-content: center;
+    margin-left: 50px;
+    margin-top: 100px;
+}
+.search1 {
+    margin-top: 50px;
+    width: 300px;
+    margin-left: 10px;
+}
+.flex-container > div {
     background-color: #f1f1f1;
     width: 100px;
     margin: 10px;
     text-align: center;
     line-height: 75px;
     font-size: 30px;
-  }
-  .flex-2-div {
-    background-color: #f1f1f1;
+}
+
+.flex-2-div {
```

Comment:
do not use classes such as flex, padding or similar, instead add proper names for the classes